src/Common/Commands.Common/PowerBISettings.cs

```diff
@@ -119,7 +119,16 @@ private string GetExecutingDirectory()
             var fileUri = new UriBuilder(codeBase);
             var directory = Uri.UnescapeDataString(fileUri.Path);
             directory = Path.GetDirectoryName(directory);
-            return directory;
+            if (string.IsNullOrEmpty(fileUri.Host))
+            {
+                return directory;
+            }
+            else
+            {
+                // Running on a fileshare
```

Comment:
Consider adding to the comment an explanation of how you are modifying the directory string.

Reply:
Will do